SYCL/InvokeSimd/ESIMD_to_SPMD_error_call_vadd_1d.cpp

```diff
@@ -0,0 +1,218 @@
+// TODO: enable on Windows once driver is ready
+// REQUIRES: gpu && linux
+// UNSUPPORTED: cuda || hip
+//
+// Check that full compilation works:
+// RUN: not %clangxx -fsycl -fno-sycl-device-code-split-esimd -Xclang -fsycl-allow-func-ptr %s -o %t.out 2>&1 | FileCheck %s
```

Comment:
could you please please move negative non-executable tests into a separate subdir?